grouper/models.py

```diff
@@ -514,7 +522,7 @@ def my_grantable_permissions(self):
                                    grantable[1] if len(grantable) > 1 else '*', ))
         return sorted(result, key=lambda x: x[0].name + x[1])
 
-    def my_groups(self):
+    def my_groups(self, ignore_enabled=False):
```

Comment:
yeah, this is essentially ignoring the fact the user is disabled. i renamed it to `ignore_user_disabled`. does that help?